sklearn/decomposition/tests/test_factor_analysis.py

```diff
@@ -83,3 +83,30 @@ def test_factor_analysis():
         precision = fa.get_precision()
         assert_array_almost_equal(np.dot(cov, precision),
                                   np.eye(X.shape[1]), 12)
+
+    # test rotation
+    n_comps = 2
+
+    results = []
+    for method in (None, "varimax", 'quartimax'):
+        fa_var = FactorAnalysis(n_components=n_comps, rotation=method)
+        results.append(fa_var.fit_transform(X))
+    for a, b in ((0, 1), (0, 2), (1, 2)):
+        assert_raises(AssertionError, assert_array_almost_equal,
```

Comment:
I think `assert not np.allclose` would be clearer

Reply:
Hm ... only here, or also in 112? 84 also uses `assert_array_almost_equal`.